README.md

```diff
@@ -13,16 +7,107 @@ which has separate instructions.
+which has separate instructions.
+
+## Choosing the Right Branch for Your Quickstart Project
+In the Arizona Quickstart Scaffolding project, different branches serve various
+purposes. Here's a guide to understanding which branch you should choose:
+
+### Main Branch (`dev-main`)
+
+- **Purpose**: The main branch contains all the latest features and changes. It
+  represents the cutting-edge development and may be less stable than specific
+  release branches. You can use it with `dev-main` or an alias for the latest
+  unreleased version (e.g., `2.8.x`).
+- **When to Use**: Choose this branch if you want access to the very latest
+  features, and you are willing to work with potentially less-tested code. Ideal
+  for development, experimentation, or testing new functionalities.
+
+### Release Branches (e.g., `2.5.x`, `2.6.x`, `2.7.x`)
+
+- **Purpose**: Release branches, named in the format of `2.x.x-dev`, are created
+  for specific versions of the project and are considered stable and
+  production-ready. They contain well-tested features suitable for live
+  environments.
+- **When to Use**: Select one of the latest two supported release branches
+  (e.g., `2.7.x-dev`, `2.6.x-dev`) if you need a stable version of the project
+  for production use, have specific environment constraints, or want to maintain
+  compatibility with particular third-party modules or a specific release.
+- **Note**: Only the latest two release branches are actively supported. Using
+  one of these ensures that you receive updates, security patches, and support
+  aligned with your system's requirements, avoiding potential conflicts and
+  achieving a robust and reliable implementation.
+- **Version Constraints in Supported Release Branches**: Each supported release
+  branch in this repository specifies a version constraint for az_quickstart in composer.json.
+  This constraint ensures that the release branch will work with compatible
+  versions of Quickstart that correspond to the compatible minor release branch,
+  providing flexibility while maintaining alignment with the intended versions.
+
+### Feature or Issue Branches
+
+Feature or issue branches in this repository are typically created to make
+updates to the scaffolding to work with specific branches of
+`az-digital/az_quickstart`. When creating such a branch, it is customary to pin
+this repository's composer.json reference for `az-digital/az_quickstart` to a
+specific issue/feature branch that the changes are intended to work with. This
+ensures that the branch's changes align precisely with a particular state of the
+Quickstart project.
+
+These branches are useful for isolated development, testing of changes, or adding new features to the scaffolding that correspond to developments in the Quickstart project.
+
+**Notes**
+- These branches are not as stable as the main or release branches, so use
```

Comment:
```suggestion
- If developing a feature for `az-digital/az_quickstart`, use the provided build tools (DDev or Lando) instead of running `composer create-project` directly.
- These branches are not as stable as the main or release branches, so use
```